### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][ColourContrast][MAS1.4.3][IntelliJ][Wipro]: Colour contrast ratio is below 4.5:1 for "Build"  under Before launch expand collapse button in Push image window.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1424]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Create a new project.
3. Navigate to and select created Project ->Azure->Run on web app.
4. "Run on web app" dialog will open.
5. Tab to "Build"  under Before launch expand collapse button and  observe the colour contrast ratio

**Expected Result:**
Colour contrast ratio should be greater than or equal to 4.5:1 for  "Build"  under Before launch expand collapse button in push image window.

**Actual Result:**
Colour contrast ratio is  4.4:1 for  "Build"  under Before launch expand collapse button in Run on web app for containers window.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)
**CCA version:** 2.5.0.0

![cca in push image](https://user-images.githubusercontent.com/22956130/35718648-5aa69272-079a-11e8-8268-1f5857b67336.png)

**Comment by Eskibear** [comment]
content of the dialog is generated from IntelliJ IDEA's template, it's IntelliJ IDEA's own widget. And accoding to IntelliJ community, the color can be changed by user's setting.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749864
